schemas_matlab/example_NewfieldUpdate.m

```diff
@@ -0,0 +1,30 @@
+% Example Script to make a new field, and update the field
+
+
+
+%% Make new field
+addAttribute(acquisition.TowersBlock, 'block_performance: float  # performance in the current block')
+
+%% update one field in TowersBlock
+allBlocks = fetch(acquisition.TowersBlock); % Get the primary keys of all Blocks
+reverseStr = '';
+for key_idx = 1:length(allBlocks)
+    key = allBlocks(key_idx);
+    trials = fetch(acquisition.TowersBlockTrial & key, '*'); % Get trials in blocks
+    correct_counter = 0;
+    for i = 1:length(trials)
+        correct_counter = correct_counter + strcmp(trials(i).trial_type, trials(i).choice);
+    end
```

Comment:
This for loop could be avoided with vector computation, which is faster. `sum(strcmp({trials.trial_type}, {trials.choice}))`